modules/placement-driver/src/integrationTest/java/org/apache/ignite/internal/placementdriver/ActiveActorTest.java

```diff
@@ -193,453 +144,22 @@ private boolean checkSingleActiveActor(String leaderName) {
-     * @return List on network addresses.
-     */
-    private static List<NetworkAddress> getNetworkAddresses(int nodes) {
-        List<NetworkAddress> addresses = IntStream.range(PORT_BASE, PORT_BASE + nodes)
-                .mapToObj(port -> new NetworkAddress("localhost", port))
-                .collect(Collectors.toList());
-        return addresses;
-    }
-
-    /**
-     * Checks the condition after cluster and raft clients initialization, waiting for this condition.
-     *
-     * @param leaderName Current leader name.
-     * @throws InterruptedException If failed.
-     */
-    private void afterInitCheckConditionWithWait(String leaderName) throws InterruptedException {
-        assertTrue(waitForCondition(() -> afterInitCheckCondition(leaderName), 10_000));
-    }
-
-    /**
-     * Checks the condition after cluster and raft clients initialization.
-     *
-     * @param leaderName Current leader name.
-     * @return Condition result.
-     */
-    private boolean afterInitCheckCondition(String leaderName) {
-        return checkSingleActiveActor(leaderName);
-    }
-
-    /**
-     * Checks the condition after leader change, waiting for this condition.
-     *
-     * @param leaderName Current leader name.
-     * @throws InterruptedException If failed.
-     */
-    private void afterLeaderChangeCheckConditionWithWait(String leaderName) throws InterruptedException {
-        assertTrue(waitForCondition(() -> afterLeaderChangeCheckCondition(leaderName), 10_000));
-    }
-
-    /**
-     * Checks the condition after leader change.
-     *
-     * @param leaderName Current leader name.
-     * @return Condition result.
-     */
-    private boolean afterLeaderChangeCheckCondition(String leaderName) {
-        return checkSingleActiveActor(leaderName);
-    }
-
-    /**
-     * Replication test group class.
-     */
-    public enum TestReplicationGroup implements ReplicationGroupId {
-        /** Replication group id. */
-        GROUP_ID;
-
-        /** {@inheritDoc} */
-        @Override
-        public String toString() {
-            return "TestReplicationGroup";
-        }
-    }
-
-    private static class TestRaftGroupListener implements RaftGroupListener {
-        @Override
-        public void onWrite(Iterator<CommandClosure<WriteCommand>> iterator) {
-            iterator.forEachRemaining(closure -> {
-                closure.result(null);
-            });
-        }
-
-        @Override
-        public void onRead(Iterator<CommandClosure<ReadCommand>> iterator) {
-        }
-
-        @Override
-        public void onSnapshotSave(Path path, Consumer<Throwable> doneClo) {
-        }
+    private static Entry emptyMetastoreEntry() {
+        Entry entry = mock(Entry.class);
 
-        @Override
-        public boolean onSnapshotLoad(Path path) {
-            return true;
-        }
+        when(entry.empty()).thenReturn(true);
 
-        @Override
-        public void onShutdown() {
-        }
+        return entry;
```

Comment:
Below this code we have `LogicalTopologyServiceTestImpl` and also we have exactly the same `org.apache.ignite.internal.topology.LogicalTopologyServiceTestImpl`. Please do something with that